migrations/sqls/20241116085102-initialize-up.sql

```diff
@@ -0,0 +1,275 @@
+-- back up with old schema
+alter schema badgehub rename to badgehub_old;
+create schema badgehub;
+create table badgehub.migrations
+(
+    like badgehub_old.migrations including all
+);
+
+-- create tables
+
+create table badges
+(
+    slug       text not null primary key,
+    name       text,
+    created_at timestamptz default now(), -- creation timestamp
+    updated_at timestamptz default now(), -- update timestamp
+    deleted_at timestamptz                -- soft delete timestamp (nullable)
+);
+
+create table users
+(
+    id                text primary key,          -- using text as recommended
+    email             text unique,               -- using text as recommended
+    admin             boolean,
+    name              text not null,
+    password          text not null,
+    remember_token    text,
+    editor            text,
+    public            boolean,
```

Comment:
What is the difference between `public` and `show_projects`?

Reply:
No Idea honestly. 
Did not look into User data at all yet except for being able to relate them to a project.
Maybe @edwinm knows?